util/constants.go

```diff
@@ -0,0 +1,5 @@
+package util
+
+const (
+	TEMPLATE_FILE_PATH string = "templates/plugins-step.tmpl"
```

Comment:
nit: a good practice is to use the CamelCase instead of CAPS for var names

Reply:
Done